Dockerfile

```diff
@@ -52,5 +55,7 @@ ENV HOME=/home/runner
 ADD utils/entrypoint.sh /bin/entrypoint
 RUN chmod +x /bin/entrypoint
 
+USER runner
```

Comment:
bikeshed

should we consider an 'ansible' user over runner? I don't have a preference myself, but I could see some folks maybe wanting it.